cli/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -412,16 +441,33 @@ mod tests {
         assert!(actual.path.is_absolute());
     }
 
-    /// Write a manifest file with invalid toml to ensure we can catch
     #[test]
     fn cache_activation_path() {
         let (flox, temp_dir) = flox_instance();
 
         let environment_temp_dir = tempfile::tempdir_in(&temp_dir).unwrap();
         let pointer = PathPointer::new("test".parse().unwrap());
 
-        let mut env =
-            PathEnvironment::init(pointer, environment_temp_dir.path(), temp_dir.path()).unwrap();
+        let mut env = PathEnvironment::init(
+            pointer,
+            environment_temp_dir.path(),
+            temp_dir.path(),
+            &flox.system,
+        )
+        .unwrap();
+        // building requires packages in the manifest, install one so the build succeeds and the outlink is built
```

Comment:
question blocking: isn't that a bug? (I should be able to e.g. have an environment that just sets some env vars). Is there a way to test the behavior you're adding here without adding test code due to the bug?

Reply:
There's a check in `buildenv` that throws an exception if there are no packages, so while this is desired behavior I'm not sure how this test ever passed. That exception is what I'm running into in CI.

Reply:
Weird. nonblocking: can you add a TODO to fix the bug and drop this?

Reply:
without lookin i think the exception is raised if there is no `packages.<system>` attribute for a specific system. The error might sound like `packages.<system>` is an empty set.